MinPlatformPkg/Acpi/AcpiTables/AcpiPlatform.c

```diff
@@ -958,6 +958,10 @@ InstallMcfgFromScratch (
   UINTN                                                                                 TableHandle;
 
   PciSegmentInfo = GetPciSegmentInfo (&SegmentCount);
+  if (PciSegmentInfo == NULL) {
+    DEBUG ((DEBUG_ERROR, "Could not create new hob for PciSegmentInfo\n"));

```

Comment:
I think the debug message needs updated.

Reply:
What would you like it updated to? I made this the message because that's the scenario in which it returns null.

Reply:
Maybe just `"Could not get PciSegmentInfo\n"`?  That does make it more confusing why it's returning `EFI_OUT_OF_RESOURCES`. Idk.

Reply:
```suggestion
    ASSERT (PciSegmentInfo != NULL);

```

I'd just assert and let assert message clearly indicate the problem.